libraries-2/src/main/java/com/baeldung/jcommander/usagebilling/service/DefaultSubmitUsageService.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.jcommander.usagebilling.service;
+
+import com.baeldung.jcommander.usagebilling.model.UsageRequest;
+
+import java.util.UUID;
+
+class DefaultSubmitUsageService implements SubmitUsageService {
+
+    DefaultSubmitUsageService() {
+    }
```

Comment:
Do we need this constructor?

Reply:
Yes, missed this while cleaning up... was using a parameterized constructor in an earlier version of this class. Removed.